### PR TITLE
Created ERD Diagram

### PR DESCRIPTION
**Pull request by emily-curtis**
Ticket #11
# Description
First draft for ERD diagram. Feel free to comment improvements.

### CODE REVIEW

File: Screenshot 2024-12-02 at 14.06.39.png
Comment (ebradley12) [inline Screenshot 2024-12-02 at 14.06.39.png:1]: All the data types are bigint - could you change them to be varchar , datetime etc

Comment (ebradley12) [inline Screenshot 2024-12-02 at 14.06.39.png:1]: We need to add subscriber ID, email address and if they are signed up to alerts

Comment (ebradley12) [inline Screenshot 2024-12-02 at 14.06.39.png:1]: Can we change 'album' table to 'release'

Comment (ebradley12) [inline Screenshot 2024-12-02 at 14.06.39.png:1]: Can we add country ID to artist and maybe remove from customer

**Review by ebradley12 — COMMENTED** [review]
Nice work Emily! This looks really good and the relationships between tables all look great.

File: erd.png
Comment (ebradley12) [inline erd.png:1]: Could we change the album_id in the sale table to be release_id for consistency :)

[the remark this answers is not in the document]
Comment (emily-curtis) [reply erd.png:1]: good shout!!!!!

**Review by GabrielNsiah — APPROVED** [review]
Looks solid to me.

**Pull request merged**